public/index.html

```diff
@@ -10,34 +10,11 @@
       content="Web site created using create-react-app"
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
```

Comment:
- [ ] The browser console is generating an error because you deleted this logo. Kindly delete all the links to the file that were deleted from the `index.html` so as to clear these errors.